vmdb/app/models/vm_microsoft.rb

```diff
@@ -7,4 +7,8 @@ def self.calculate_power_state(raw_power_state)
     else                        super
     end
   end
+
+  def supports_migrate?
+    false
+  end
```

Comment:
Actually, since you are putting true in the base class, is migrate available for template on *any* provider?  What does that even mean?